src/RBCXMpu.cpp

```diff
@@ -0,0 +1,193 @@
+                      + m_mpuMotion.accel.x * m_mpuMotion.accel.x))
+        * RAD_2_DEG; // [-180°,+180°]
+
+    m_mpuMotion.angleAcc.y
+        = -atan2(m_mpuMotion.accel.x,
+              sqrt(m_mpuMotion.accel.z * m_mpuMotion.accel.z
+                  + m_mpuMotion.accel.y * m_mpuMotion.accel.y))
+        * RAD_2_DEG; // [- 90°,+ 90°]
+
+    unsigned long Tnew = xTaskGetTickCount() * portTICK_RATE_MS;
+    m_interval = (Tnew - m_preInterval) * 1e-3;
+    m_preInterval = Tnew;
+
+    // Correctly wrap X and Y angles (special thanks to Edgar Bonet!)
+    // https://github.com/gabriel-milan/TinyMPU6050/issues/6
+    m_mpuMotion.angle.x
+        = wrap((m_mpuMotion.angleAcc.x
+                   + wrap(m_mpuMotion.angle.x + m_mpuMotion.gyro.x * m_interval
+                           - m_mpuMotion.angleAcc.x,
+                       180))
+                + m_mpuMotion.angleAcc.x,
+            180);
+    m_mpuMotion.angle.y = wrap(
+        (m_mpuMotion.angleAcc.y
+            + wrap(m_mpuMotion.angle.y + sgZ * m_mpuMotion.gyro.y * m_interval
+                    - m_mpuMotion.angleAcc.y,
+                90))
+            + m_mpuMotion.angleAcc.y,
+        90);
+
+    // angleZ += gyroZ*m_interval; // not wrapped (to do???)
+    m_mpuMotion.angle.z
+        += m_mpuMotion.gyro.z * m_interval; // not wrapped (to do???)
+}
+
+/* Wrap an angle in the range [-limit,+limit] (special thanks to Edgar Bonet!) */
+float Mpu::wrap(float angle, float limit) {
+    while (angle > limit)
+        angle -= 2 * limit;
+    while (angle < -limit)
+        angle += 2 * limit;
+    return angle;
+}
+
+MpuVector Mpu::getAcc() { return m_mpuMotion.accel; }
+float Mpu::getAccX() { return m_mpuMotion.accel.x; }
+float Mpu::getAccY() { return m_mpuMotion.accel.y; }
+float Mpu::getAccZ() { return m_mpuMotion.accel.z; }
+
+MpuVector Mpu::getGyro() { return m_mpuMotion.gyro; }
+float Mpu::getGyroX() { return m_mpuMotion.gyro.x; }
+float Mpu::getGyroY() { return m_mpuMotion.gyro.y; }
+float Mpu::getGyroZ() { return m_mpuMotion.gyro.z; }
+
+MpuVector Mpu::getAngle() { return m_mpuMotion.angle; }
+float Mpu::getAngleX() { return m_mpuMotion.angle.x; }
+float Mpu::getAngleY() { return m_mpuMotion.angle.y; }
+float Mpu::getAngleZ() { return m_mpuMotion.angle.z; }
+
+long Mpu::getInterval() { return m_interval; }
+
+void Mpu::setCompressCoef(uint8_t coef) {
+    sendMpuReq(
+        CoprocReq_MpuReq { .which_mpuCmd = CoprocReq_MpuReq_setCompressCoef_tag,
+            .mpuCmd = {
+                .setCompressCoef = coef,
+            } });
+}
+
+uint8_t Mpu::getCompressCoef() { return m_compressCoef; }
+
+void Mpu::sendMpuReq(CoprocReq_MpuReq mpuReq) {
+    rb::Manager::get().sendToCoproc(CoprocReq {
+        .which_payload = CoprocReq_i2cReq_tag,
+        .payload = { .i2cReq = { .which_payload = CoprocReq_I2cReq_mpuReq_tag,
+                         .payload = {
+                             .mpuReq = mpuReq,
+                         } } } });
```

Comment:
Nitpick a tip: když se dá za/před `}` čárka, tak je to validní kód, a clang-format to pak nerozhaší tímto způsobem, tu inicializaci structu, používám to třeba tady https://github.com/RoboticsBrno/RB3204-RBCX-library/blob/846faf0efe643d6272c18d9472ab50c7e6ea5e7c/src/RBCXLeds.cpp#L32-L39